packages/presentation/lib/screen/cast_list/cast_crew_data.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:domain/model/people_and_images_model.dart';
+
+class CastCrewData {
+  CastCrewData({
+    required this.detailsAboutPeople,
+  });
+
+  final List<PeopleAndImagesModel>? detailsAboutPeople;
+
+  factory CastCrewData.init() => CastCrewData(
+        detailsAboutPeople: [],
+      );
```

Comment:
no need to make detailsAboutPeople nullable

Reply:
so how can I do without it?

Reply:
where do u use nullable? where it can be null? when u init this class u using List.empty. When u  do .copyWith() u also no need this variable nullable

Reply:
it's ok to have the list nullable by default I think